cmd/internal/account/before_each.go

```diff
@@ -18,11 +18,20 @@ var (
 )
 
 func beforeEach(cliCtx *cli.Context) error {
-	if nArgs := cliCtx.NArg(); nArgs != 2 {
-		return fmt.Errorf("expecting account alias as the only argument, got %d instead", nArgs)
+	var nArgs int
```

Comment:
I kinda wonder if it would make sense to allow each subcommand to specify their required arguments, but for now I just went with this approach since I'm still a Go noob, so I didn't want to complicate things.

Reply:
Yes, I think that's the way to go. I'll fix this in my PR instead. I just tried to be too clever there.